uefi/src/proto/device_path/device_path_gen.rs

```diff
@@ -3,7 +3,7 @@
 // This file was automatically generated with:
 // `cargo xtask gen-code`
 //
-// See //xtask/src/device_path/README.md for more details.
```

Comment:
does this `//` syntax has a special meaning or was it a mistake?

Reply:
I'm pretty sure it was a mistake. Could you also make the path be within `` ` `` while we're at it?

Reply:
This was actually intentional in that `//` is sometimes used as a convention to indicate the root of a repository. But I absolutely don't feel strongly about it so please feel free to change it :)

Reply:
> This was actually intentional in that // is sometimes used as a convention to indicate the root of a repository

Interesting, I never saw this. For me, `/` is always the root in the current context